src/stories/drawer/Drawer.tsx

```diff
@@ -338,36 +325,46 @@ export const Drawer = <T extends string[] | readonly string[] = string[]>({
                                         >
                                             Close dialog
                                         </Button>
-                                    </div>
-                                </RemoveFromDOM>
+                                    </RemoveFromDOM>
```

Comment:
I'm not personally testing all of the variants here where these `RemoveFromDOM`s and `VisuallyHidden`s should be functioning but please double check they work as expected

Reply:
Yes all the permutations seem to work